lms/static/js/student_profile/views/learner_profile_view.js

```diff
@@ -1,71 +1,77 @@
+                    } else {
+                        return this.options.accountSettingsModel.get('profile_is_public');
+                    }
+                },
 
-            render: function () {
-                this.$el.html(_.template(learnerProfileTemplate)({
-                    username: this.options.accountSettingsModel.get('username'),
-                    ownProfile: this.options.ownProfile,
-                    showFullProfile: this.showFullProfile()
-                }));
-                this.renderFields();
-                return this;
-            },
+                render: function() {
+                    HtmlUtils.setHtml(
+                        this.$el,
+                        HtmlUtils.template(learnerProfileTemplate)({
+                            username: this.options.accountSettingsModel.get('username'),
+                            ownProfile: this.options.ownProfile,
+                            showFullProfile: this.showFullProfile()
+                        })
+                    );
+                    this.renderFields();
+                    return this;
+                },
 
-            renderFields: function() {
-                var view = this;
+                renderFields: function() {
+                    var view = this;
 
-                if (this.options.ownProfile) {
-                    var fieldView = this.options.accountPrivacyFieldView,
-                        settings = this.options.accountSettingsModel;
-                    fieldView.profileIsPrivate =  !settings.get('year_of_birth');
-                    fieldView.requiresParentalConsent = settings.get('requires_parental_consent');
-                    fieldView.isAboveMinimumAge = settings.isAboveMinimumAge();
-                    fieldView.undelegateEvents();
-                    this.$('.wrapper-profile-field-account-privacy').append(fieldView.render().el);
-                    fieldView.delegateEvents();
-                }
+                    if (this.options.ownProfile) {
+                        var fieldView = this.options.accountPrivacyFieldView,
+                            settings = this.options.accountSettingsModel;
+                        fieldView.profileIsPrivate =  !settings.get('year_of_birth');
+                        fieldView.requiresParentalConsent = settings.get('requires_parental_consent');
+                        fieldView.isAboveMinimumAge = settings.isAboveMinimumAge();
+                        fieldView.undelegateEvents();
+                        this.$('.wrapper-profile-field-account-privacy').append(fieldView.render().el);
+                        fieldView.delegateEvents();
+                    }
 
-                this.$('.profile-section-one-fields').append(this.options.usernameFieldView.render().el);
+                    this.$('.profile-section-one-fields').append(this.options.usernameFieldView.render().el);
```

Comment:
Maybe the linter should allow append with anything that ends with ".el"?  Of course if this is put into a variable beforehand, that would break it.